### PR TITLE
link to codelcimate docs in readme

### PR DESCRIPTION
**Pull request by ABaldwinHunter**
cc @codeclimate/review

### CODE REVIEW

**Comment by wfleming** [comment]
nice :+1: 

FWIW, you might want to extract the docs URL to a named markdown URL at the bottom of the page or something. If we ever ended up changing that URL for any reason, there's an excellent chance we'd remember to change one of these & not the other.

I believe the syntax works like this:

```markdown
[RuboCop engine documentation][cc-docs-rubocop]



[cc-docs-rubocop]: https://docs.codeclimate.com/docs/rubocop
```

File: README.md [hunk 1]
```diff
@@ -6,6 +6,8 @@
 
 RuboCop helps you enforce many of the guidelines outlined in the community [Ruby Style Guide](https://github.com/bbatsov/ruby-style-guide). Most aspects of its behavior can be tweaked via various [configuration options](https://github.com/bbatsov/rubocop/blob/master/config/default.yml), which are set in a **.rubocop.yml** file.
 
+You can find some basic setup instructions and links to the RuboCop OSS project below. For additional Code Climate-specific config details, check out our [RuboCop engine documentation here](https://docs.codeclimate.com/docs/rubocop).
```
Comment (wfleming) [inline README.md]: Personal opinion, but I think "here" is redundant in the link text. It's a link, and it's "RuboCop engine documentation": I don't think more is required.

Comment (ABaldwinHunter) [inline README.md]: :+1:  sounds good

**Comment by ABaldwinHunter** [comment]
@wfleming thanks great idea!

Added link

**Comment by wfleming** [comment]
:shipit:

**Pull request merged**